src/problem/util/texGenerator.ts

```diff
@@ -0,0 +1,28 @@
+export const gcd = (
```

Comment:
the content in this file looks nothing like tex generator ;-)

Reply:
On working...